src/components/Editor/QuestionTypeDropdown.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div>
+    <div :class="selectedOptionClass">
+      <div class="relative">
+        <!-- selected option -->
+        <button
+          type="button"
+          @click="toggleDropdownDisplay"
+          :disabled="isDisabled"
+          class="disabled:opacity-50 w-full flex space-x-2 bg-primary rounded-lg shadow-lg p-2 text-left cursor-default focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-primary sm:text-sm items-center"
+        >
+          <span class="flex flex-1 space-x-2 items-center">
+            <inline-svg
+              :src="getIconSource(selectedOption.icon)"
+              class="text-white h-4 w-full bp-420:w-2/3 bp-500:w-full sm:w-2/3 fill-current"
+            ></inline-svg>
+          </span>
+          <!-- dropdown icon -->
+          <inline-svg
+            :src="require('@/assets/images/chevron-down-solid-white.svg')"
+            class="h-4 w-4 text-primary fill-current"
```

Comment:
can add a class here for the dropdown arrow to move (like it happens in the table sorting column) - not necessary though
```js
dropdownIconClass() {
    return [
     { "transform rotate-180": this.showDropdown },
     "transition ease duration-800",
    ]
}
```